tests/v2/validation/nodescaling/README.md

```diff
@@ -82,7 +57,7 @@ scalingInput:
   gkeNodePool:
     initialNodeCount: 1
 ```
-NOTE: When scaling AKS and EKS, you will need to make sure that the `maxCount` and `maxSize` parameter is greater than the desired scale amount, respectively. For example, if you wish to have 6 total EKS nodes, then the `maxSize` parameter needs to be at least 7. This is not a limitation of the automation, but rather how EkS specifically handles nodegroups.
+NOTE: When scaling AKS and EKS, you will need to make sure that the `maxCount` and `maxSize` parameter is greater than the desired scale amount, respectively. For example, if you wish to have 6 total EKS nodes, then the `maxSize` parameter needs to be at least 7. This is not a limitation of the automation, but rather how EKS specifically handles nodegroups.
```

Comment:
What about an autoscaling-enabled option? IIRC, it was important for AKS, right? And I think with option autoscaling-disabled tests are working with the shown `scalingInput`. But for `maxCount` we need autoscaling-enabled, and the shown configuration won't be enough and needs the full configuration like provisioning tests, right?

I don't know how, but I think we need that info here

Reply:
I don't have an issue putting that here, I went with less is more here, but I'll add it.

Reply:
Resolved.